controllers/UserController.js

```diff
@@ -0,0 +1,44 @@
+const apiResponse = require("../helpers/apiResponse");
+const UserModel = require("../models/UserModel");
+const { validationResult } = require("express-validator");
+const auth = require("../middlewares/jwt");
+
+exports.getUser = [
+  auth,
+  (req, res) => {
+    try {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 4 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
		try {
```